.github/flake8-matchers.json

```diff
@@ -0,0 +1,32 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "flake8-error",
+            "severity": "error",
+            "pattern": [
+                {
+                    "regexp": "^([^:]*):(\\d+):(\\d+): ([EF]\\d\\d\\d) (.*)$",
+                    "file": 1,
+                    "line": 2,
+                    "column": 3,
+                    "code": 4,
+                    "message": 5
+                }
+            ]
+        },
+        {
+            "owner": "flake8-warning",
+            "severity": "warning",
+            "pattern": [
+                {
+                    "regexp": "^([^:]*):(\\d+):(\\d+): ([W]\\d\\d\\d) (.*)$",
+                    "file": 1,
+                    "line": 2,
+                    "column": 3,
+                    "code": 4,
+                    "message": 5
+                }
+            ]
+        }
+    ]
+}
```

Comment:
How will these matchers react to the error codes of `flake8-annotations`, ANN001? They don't match either pattern, as your pattern for errors only matched `[EF]` in the error code section and your warning one only `[W]`. A slightly less sophisticated method would be to have `flake8` output its error messages in the proper format for annotations directly. 

This means you wouldn't have to worry about the CI when error codes change in the future (e.g., we add more plugins).